cmd/snap/cmd_snap_op.go

```diff
@@ -842,9 +842,10 @@ func (x *cmdRefresh) Execute([]string) error {
 		x.setModes(opts)
 		return x.refreshOne(names[0], opts)
 	}
-	// transactional flag is the only one with meaning when
+	// transactional and ignore-running flags are the only ones with meaning when
 	// refreshing many snaps
 	opts := &client.SnapOptions{
+		IgnoreRunning: x.IgnoreRunning,
```

Comment:
The `x.IgnoreRunning` check wasn't covered before so I guess we should add a test to cover this

Reply:
Added